src/common/utils/setup-helpers.js

```diff
@@ -306,7 +306,7 @@ function configureDelayAdsInitialization(options: KPOptionsObject): void {
  * @returns {void}
  */
 function checkNativeTextTracksSupport(options: KPOptionsObject): void {
-  if (isSafari()) {
+  if (isIos()) {
```

Comment:
```suggestion
  if (isSafari() || isIos()) {
```
do include safari desktop as well